packages/lwc/README.md

```diff
@@ -0,0 +1,73 @@
+Lightning Web Components (LWC) is a blazing fast, enterprise-grade web components foundation for building user interfaces. LWC is a thin layer around the low-level Web Components APIs. The `lwc` package is the main entry point for dependencies.
```

Comment:
How about: LWC provides a simple authoring format for UI Components. LWC compiles this authoring format into low-level Web Component APIs.

Reply:
This is for the Second Sentence.